### PR TITLE
Re-enable cliff impacts

### LINKED ISSUE #1843: Re-enable cliff impacts

### PR DESCRIPTION
**Pull request by anth-volk**
## Description

Fixes #1843. To not crash, this requires the merging of https://github.com/PolicyEngine/policyengine-api/pull/1982; to correctly calculate UK results, https://github.com/PolicyEngine/policyengine-uk/pull/997 must also be merged. This PR will remain in draft until the merging of the relevant API changes and until I can capture some screenshots.

## Changes

Re-enables cliff impacts. Also removes a hanging semicolon that created a floating semicolon on the cliff impacts output page. Most of this code merely un-comments code that was previously commented out as a result of https://github.com/PolicyEngine/policyengine-app/pull/1099.

## Screenshots

Screen Shot 2024-11-19 at 1 09 56 AM
Screen Shot 2024-11-19 at 1 11 49 AM

## Tests

N/A

### CODE REVIEW

File: src/pages/policy/output/FetchAndDisplayImpact.jsx [hunk 1]
```diff
@@ -283,24 +272,18 @@ export function FetchAndDisplayCliffImpact(props) {
     return <DisplayError error={error} />;
   }
 
-  // Remove the below block when cliff impacts are reinstated
+  // Remove the below commented block when cliff impacts are reinstated
```
Comment (nikhilwoodruff) [inline src/pages/policy/output/FetchAndDisplayImpact.jsx:275]: Can we remove this?

**Review by nikhilwoodruff — APPROVED** [review]

**Comment by anth-volk** [comment]
Good to go now

**Comment by MaxGhenis** [comment]
What does it say the baseline cliff rate and gap are, by CPS and ECPS?

**Comment by anth-volk** [comment]
At the moment, we don't actually permit the running of baseline sims, and thus I cannot run the cliff impacts, either. Is there another policy we can use?

Alternatively: should we be enabling baseline runs?

**Comment by MaxGhenis** [comment]
Just hover over the bar chart

**Comment by anth-volk** [comment]
Here are screenshots of the requested four outputs:
Screen Shot 2024-11-19 at 11 21 10 PM
Screen Shot 2024-11-19 at 11 21 18 PM
Screen Shot 2024-11-19 at 11 33 37 PM
Screen Shot 2024-11-19 at 11 33 44 PM

**Comment by anth-volk** [comment]
This is using the MTR-based definition and a delta of $1,000

**Comment by MaxGhenis** [comment]
Thanks. For posterity (data for 2024):

| Dataset | Cliff rate | Cliff gap |
| - | - | - |
| CPS | 3.5% | $51.3bn |
| ECPS | 4.3% | $70.3bn |

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**